src/corehost/cli/args.cpp

```diff
@@ -103,6 +109,16 @@ bool parse_arguments(const pal::string_t& deps_path, const pal::string_t& probe_
     }
 
     pal::getenv(_X("DOTNET_PACKAGES_CACHE"), &args.dotnet_packages_cache);
```

Comment:
We should rename DOTNET_PACKAGES_CACHE to something like DOTNET_HOSTING_OPTIMIZATION_CACHE. @leecow @blackdwarf  What do you think?